docker/1.4.0/final/Dockerfile.eia

```diff
@@ -0,0 +1,36 @@
+ARG py_version
+
+FROM awsdeeplearningteam/mxnet-model-server:base-cpu-py$py_version
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends \
+    libopencv-dev \
```

Comment:
why libopencv again in EIA dockerfile, looks like this part is being copied across multiple Dockerfile.